pipeline/runtime/common.go

```diff
@@ -83,3 +86,23 @@ func fetchOutputVariables(outputFile string, out io.Writer) (map[string]string,
 	}
 	return outputs, nil
 }
+
+// setTiEnvVariables sets the environment variables required for TI
+func setTiEnvVariables(step *spec.Step, config *api.TIConfig) {
+
+	if config == nil {
+		return
+	}
+	if step.Envs == nil {
+		step.Envs = map[string]string{}
+	}
+
+	envMap := step.Envs
+	envMap[ti.TiSvcEp] = config.URL
+	envMap[ti.TiSvcToken] = config.Token
+	envMap[ti.AccountIDEnv] = config.AccountID
+	envMap[ti.OrgIDEnv] = config.OrgID
+	envMap[ti.ProjectIDEnv] = config.ProjectID
+	envMap[ti.PipelineIDEnv] = config.PipelineID
+	envMap[ti.InfraEnv] = "VM"
```

Comment:
Use an enum declared in a common location.

Reply:
This repo does not seem to import harness-core and has all the structs redefined. Check types.go in this repo. Hence I did not create an enum in a common location.